ethree-common/src/main/java/com/virgilsecurity/android/common/worker/StreamsEncryptWorker.kt

```diff
@@ -94,6 +95,27 @@ internal class StreamsEncryptWorker internal constructor(
         return decryptInternal(inputStream, outputStream, card.publicKey)
     }
 
+    internal fun encryptShared(inputStream: InputStream,
+                               inputStreamSize: Int,
+                             outputStream: OutputStream): ByteArray {
+        if (inputStream.available() == 0) throw EmptyArgumentException("inputStream")
```

Comment:
Empty streams are allowed for encryption

Reply:
Fixed